README.md

```diff
@@ -9,18 +9,35 @@ code.
 
 We compare performance of the R model against 1. a model written
 entirely in C++ using the C++ components of `individual` and 2. an
-`individual` R model that uses custom "prefab" C++ processes.
+`individual` R model that uses custom “prefab” C++ processes.
 
 ### Closed SIR Model
 
     ##   model_version median_time_seconds speedup
-    ## 1             R          0.73188955     1.0
-    ## 2             C          0.03573859    20.0
-    ## 3        Prefab          0.22616319     3.2
+    ## 1             R          0.59080680     1.0
+    ## 2             C          0.02948761    20.0
+    ## 3        Prefab          0.18778400     3.1
 
 ### Open SIR Model
 
     ##   model_version median_time_seconds speedup
-    ## 1             R           1.3406097     1.0
-    ## 2             C           0.5096369     2.6
-    ## 3        Prefab           0.7546200     1.8
+    ## 1             R           1.1542080     1.0
+    ## 2             C           0.3327583     3.5
```

Comment:
only a slight speedup here (numbers are all a little lower than before because I ran this on my desktop, but the ratios remained the same aside from this one!)